### DISCUSSION TITLE
Обсуждение функционала медиа "Разрабы"

### DISCUSSION THREAD

**Discussion by philippranzhin** [razrabs-media/journal#355] [General]
Здесь предлагаю обусждать новые фичи, изменение поведения и прочие пожелания сообщества к медиа

**Comment by GNMKR** [comment]
Может на Issues асайнить исполнителей? Я вот пару часов потратил на issue, думал вливать, а оказалось, что по тому же issue уже висит PR
:(

**Reply by philippranzhin** [reply]
кстати, серьезная проблема да — так и будем делать

**Comment by wadzari** [comment]
В мобилках можно высчитывать высоту изображений, чтоб избежать скачков layout при рендере страницы
<details>
  <summary>GIF</summary>
  
![Peek 2022-08-08 11-39](https://user-images.githubusercontent.com/34591001/183377410-00c1545c-28a2-4db7-ba64-b4c26c055b79.gif)
</details>

**Reply by philippranzhin** [reply]
Мы пока не можем, они же у нас на гитхабе лежат, скоро завезём свое хранилище — так и будем делать и на мобилках и на десктопе
